Sources/StreamChatUI/MessageActionsPopup/ChatMessagePopupVC.swift

```diff
@@ -6,10 +6,17 @@ import StreamChat
 import UIKit
 
 /// `ChatMessagePopupVC` is shown when user long-presses a message.
-/// By default, it has a blurred background, reactions, and actions which are shown for a given message
-/// and with which user can interact.
+/// By default, it has a blurred background, a reactions picker at the top, the message in the center,
+/// and at the bottom the message actions sheet or the reaction authors list.
 open class ChatMessagePopupVC: _ViewController, ComponentsProvider {
-    /// `ContainerStackView` encapsulating underlying views `reactionsController`, `actionsController` and `messageContentView`.
+    /// The scroll view which contains the content view of the popup.
+    open private(set) var scrollView = UIScrollView().withoutAutoresizingMaskConstraints
+
+    /// The view that contains all views and is responsible the make the popup view scrollable.
+    open private(set) var contentView = UIView().withoutAutoresizingMaskConstraints
```

Comment:
The `messageContainerStackView` is the only subview of `contentView`. Why don't we add `messageContainerStackView` to scroll view directly?

Reply:
I tried that, but probably because of some issues on ContainerStackView, it was causing weird glitches :/

Reply:
Ok, let's keep it then (with accessibility id)👍